cdk/src/open-data-platform/data-plane/data-import/handler-factory.ts

```diff
@@ -1,108 +1,199 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
-import { ConnectionPool } from '@databases/pg';
 import * as AWS from 'aws-sdk';
-import { connectToDb } from '../schema/schema.handler';
+import { AnyLengthString } from 'aws-sdk/clients/comprehendmedical';
 
 // These libraries don't have types, so they are imported in a different way.
 const { chain } = require('stream-chain');
 const { streamArray } = require('stream-json/streamers/StreamArray');
 const Batch = require('stream-json/utils/Batch');
 const Pick = require('stream-json/filters/Pick');
 
+export interface ImportRequest {
```

Comment:
same comment here as below - since the handler is not insert-specific, could we just call this geojsonparsingrequest?

Reply:
Good point. I've removed the "insert" or "import" language and replaced it with "process".